share/spice/whois/whois.js

```diff
@@ -113,16 +115,25 @@
         return api_result.dataError && api_result.dataError === 'MISSING_WHOIS_DATA';
     }
 
-    //Converts timestamp into local time using getDateFromString
+    //Converts timestamp into local time using moment.js
     function prettifyTimestamp(timestamp) {
         if(!timestamp) { return; }
-        var dateObj = DDG.getDateFromString(timestamp),
-            monthArr = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"],
-            day = dateObj.getDate(),
-            month = monthArr[dateObj.getMonth()+1],
-            year = dateObj.getFullYear();
-        if(day<10) {day='0'+day};
-        return month + " " + day + ", " + year;
+
+        //When using the expiresDateNormalized (example: 2020-09-14 00:00:00 UTC); need to use custom date formatting
+        //(if !api_result.updatedDate && !api_result.expiresDate) 
+
+        var customDate = moment(timestamp, "YYYY-MM-DD HH:mm Z"), // formatting date: 2020-09-14 00:00:00 UTC
```

Comment:
Small nit, but it seems like we're doing extra work in creating the `customDate` before we know if we'll actually need it? Let's create this in the `else` to fix that.